JPS_Ontology/ontology/ontoems/OntoEnvironmentalMeasurementStation.csv

```diff
@@ -47,8 +49,11 @@ DirectHorizontalIrradiance,Class,http://purl.org/iot/vocab/m3-lite#SolarRadiatio
 DiffuseHorizontalIrradiance,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Irradiance,IS-A,,,,The horizontal component of diffuse solar irradiance (irradiance that is scattered by the atmosphere).,https://www.theworldavatar.com/kg/ontoems/,Diffuse horizontal irradiance
 DiffuseHorizontalIrradiance,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#SolarIrradiance,IS-A,,,,The horizontal component of diffuse solar irradiance (irradiance that is scattered by the atmosphere).,https://www.theworldavatar.com/kg/ontoems/,Diffuse horizontal irradiance
 DiffuseHorizontalIrradiance,Class,http://purl.org/iot/vocab/m3-lite#SolarRadiation,IS-A,,,,The horizontal component of diffuse solar irradiance (irradiance that is scattered by the atmosphere).,https://www.theworldavatar.com/kg/ontoems/,Diffuse horizontal irradiance
+DirectNormalIrradiance,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Irradiance,IS-A,,,,The irradiance arriving in a direct line from the sun as measured on a surface held perpendicular to the sun,https://www.theworldavatar.com/kg/ontoems/,Direct normal irradiance
```

Comment:
I initially drafted the ontology based on the Solcast glassary:
https://articles.solcast.com.au/en/articles/2790853-historical-data-glossary-of-terms

Having screened further data, do you believe it is worthwhile keeping `Direct Horizontal Irradiance (EBH)`? If so, we might want to refine the `includedIn` relationship according to the equation as shown on the MIRO board (DNI - includedIn - EBH - includedIn - GHI). Otherwise, we might replace `EBH` with `DNI`. Most sources only seem to focus on GHI, DHI, and DNI.
https://en.wikipedia.org/wiki/Solar_irradiance
https://pvpmc.sandia.gov/modeling-steps/1-weather-design-inputs/irradiance-and-insolation-2/global-horizontal-irradiance/

Reply:
I think we should keep direct normal irradiance only and get rid of direct horizontal irradiance, since, as you mentioned, most sources focus on DNI and not EBH. I have also came to the same observation when researching weather data.

The two weather APIs (OpenWeather, https://openweathermap.org/; Open-Meteo, https://open-meteo.com/) I have looked at only provide direct normal irradiance and not direct horizontal irradiance. Furthermore, weather files (.epw files) only store direct normal radiation and does not store direct horizontal radiation.

Reply:
> I think we should keep direct normal irradiance only and get rid of direct horizontal irradiance, since, as you mentioned, most sources focus on DNI and not EBH. I have also came to the same observation when researching weather data.
> 
> The two weather APIs (OpenWeather, https://openweathermap.org/; Open-Meteo, https://open-meteo.com/) I have looked at only provide direct normal irradiance and not direct horizontal irradiance. Furthermore, weather files (.epw files) only store direct normal radiation and does not store direct horizontal radiation.

Then please update accordingly, sounds good to me. Maybe we could even include the equation into the description(s) to make them consistent

Reply:
Updated with equation in descriptions.